docs/make.jl

```diff
@@ -1,8 +1,122 @@
+            cmd_upload = `asciinema upload $(path_record)`
+
+            # Dry run not to record precompilation
+            run(cmd_jl)
+
+            # Record the REPL output with asciinema
+            run(cmd_record)
+
+            # Upload the record and get the ID
+            c = IOCapture.capture() do
+                run(cmd_upload)
+            end
+            id_upload_begin = findfirst("https://asciinema.org/a/", c.output)[end] + 1
+            id_upload_end = findnext("\n", c.output, id_upload_begin)[begin] - 1
+            id_upload = c.output[id_upload_begin:id_upload_end]
+            url_upload_begin = findfirst("https://asciinema.org/a/", c.output)[begin]
+            url_upload_end = findnext("\n", c.output, url_upload_begin)[begin] - 1
+            url_upload = c.output[url_upload_begin:url_upload_end]
+            @info "The recorded asciicast was uploaded to $(url_upload)"
+
+            # Add an markdown subsection of the example
+            script_md *= """
+
+            ### $(title)
+            Script in [`$(path_script)`]($(url_script)):
+            ```julia
+            $(read(path_script, String))
+            ```
+            Replay the script in the REPL:
+            ```
+            $(cmd_jl_str)
+            ```
+            The output will be like this:
+            ```@raw html
+            <script src="https://asciinema.org/a/$(id_upload).js" id="asciicast-$(id_upload)" async></script>
+            ```
+            """
+            push!(names_example, name)
+        end
+    end
+    write(path_md, script_md)
+    for name in setdiff(readdir(dir_examples), names_example)
+        @warn "$name is not generated"
+    end
+end
+
+sections = [
+    "Basics" => [
+        (name="readme", title="Basic example in the README", options=``),
+        (name="helloworld", title="Hello world", options=``),
+    ],
+    "More in REPL" => [
+        (name="helpmode", title="Help mode", options=``),
+        (name="pkgmode", title="Package mode", options=``),
+        (name="shellmode", title="Shell mode", options=``),
+        (name="tab_completion", title="Tab completion", options=``),
+    ],
+    "CLI options" => [
+        (name="disable_color", title="Disable color", options=`--color=no`),
+        (name="quietmode", title="Quiet mode", options=``),
+    ],
+    "Working with other packages" => [
+        (name="ohmyrepl", title="OhMyREPL.jl", options=``),
+        (name="unicodefun", title="UnicodeFun.jl", options=``),
+        (name="pythoncall", title="PythonCall.jl", options=``),
+        (name="unicodeplots", title="UnicodePlots.jl", options=``),
+        (name="unicodeplots_animated", title="UnicodePlots.jl (animated)", options=``),
+        (name="imageinterminal", title="ImageInTerminal.jl", options=``),
+        # Sixel output is not supported by asciinema
+        # (name="sixel", title="Sixel.jl", options=``),
+        # (name="plots_with_sixel", title="Sixel.jl", options=``),
```

Comment:
I have not added these examples with Sixel.jl. I guess this can be fixed with [some python script](https://github.com/AtelierArith/Replay.jl/issues/23#issuecomment-1013868142), but I would like to separate this fix into another PR.

Reply:
> separate this
I agree. I will record the screen running example about Sixel.jl and paste it into the issue https://github.com/AtelierArith/Replay.jl/issues/1 . Then I will update our documentation (in another PR).

Reply:
I agree:+1: (sorry for the bothering comment on #1)